plugins/lookup/passwordstore.py

```diff
@@ -223,6 +262,11 @@ def parse_params(self, term):
                     self.paramvals['length'] = int(self.paramvals['length'])
                 else:
                     raise AnsibleError("{0} is not a correct value for length".format(self.paramvals['length']))
+            if self.paramvals['lock'] not in ['readwrite', 'write', 'none']:
```

Comment:
This condition can never be true, since you use `self.get_option` to retrieve the value of `lock`.

Reply:
@felixfontein couldn’t it get overridden by additional terms in the lookup later (run iterates over terms)?